app/models/service.rb

```diff
@@ -238,6 +239,12 @@ def tenant_identity
     user
   end
 
+  def chargeback_report
+    miq_report = MiqReportResult.find_by(:name => chargeback_report_name)
```

Comment:
Small nit, but, I think this variable should be named `result` or `report_result`